torch/_dynamo/variables/lists.py

```diff
@@ -451,15 +448,18 @@ def __init__(self, items, **kwargs):
         else:
             raise AssertionError()
 
-        # Avoids a .item() call in the tensor slice that would attempt to get a
-        # value out fake tensors, and which would determine the output shape of
-        # the slice.  It is a workaround until
-        # https://github.com/pytorch/pytorch/pull/83567 is landed and there is
-        # more complete support for breaking on data dependent operators.
-        if not config.capture_scalar_outputs:
-            for limit in (start, stop, step):
-                if isinstance(limit, (variables.TensorVariable, DynamicShapeVariable)):
-                    unimplemented("Dynamic slicing not supported")
+        for limit in (start, stop):
+            if isinstance(limit, (variables.TensorVariable)):
+                unimplemented(
+                    "Dynamic slicing on data-dependent value is not supported"
+                )
+
+        # TODO (tugsbayasgalan) Address this later, the implementation
+        # of slice unpacking does some adjustments based on step size
+        # to start and stop. It is bit unclear what those adjustment
+        # be when step is symbolic
```

Comment:
> what those adjustments be 

did you forget a word?

Reply:
oops this is addressed. Let me publish a new version.